### PR TITLE
Make FormSelectField's fullWidth prop customizable

### PR DESCRIPTION
**Pull request by ash3rz**

### CODE REVIEW

**Review by sri2k1us — APPROVED** [review]
LGTM :shipit:

File: src/util/FormField.js [hunk 1]
```diff
@@ -257,6 +258,10 @@ const FormSelectField = ({
     );
 };
 
+FormSelectField.defaultProps = {
+    fullWidth: true,
```
Comment (psarando) [inline src/util/FormField.js:262]: 👍

File: README.md [hunk 2]
```diff
@@ -1,68 +1,39 @@
...
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `npm run eject`
-
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+### `npm run test`
```
Comment (psarando) [inline README.md]: This command can remain as simply `npm test`.
The `test` and `start` scripts are treated special by `npm` and don't require `run`.
Our other scripts, such as `npm run storybook`, still do.

**Review by psarando — APPROVED** [review]
LGTM 👍

**Comment by sri2k1us** [comment]
Looks good!

**Comment by ash3rz** [comment]
Thanks for the reviews!

**Pull request merged**